docs/ECS Fargate Tutorials/typeorm.md

```diff
@@ -241,10 +242,27 @@ aws ecr-public batch-delete-image \
 npx typeorm migration:revert
 ```
 
-3. Apply the changes described in the hosted db to your cloud account
+3. The `down` part of the second, and last, migration is called which reverts the changes and calls the `iasql_apply` function:
 
-```bash
-$ iasql apply startup
-✔ Press enter to confirm apply · yes
+```sql title="ecs-fargate-examples/typeorm/infra/1646683871219-Initial.js"
```

Comment:
It might be a good idea to do a similar change to the files here? The reasoning behind the git subdirectory filter is that it *actually* leaves the repo in a state that they can use as the basis for their own project and push to their own origin.